### DISCUSSION TITLE
Data standards - general

### DISCUSSION THREAD

**Discussion by philhb** [UNMigration/HTCDS#12] [Ideas]
Add any new general ideas or thoughts about the data standards.
For example, how can they be improve them to be more relevant in different countries? How can we improve adoption? 
How can the documentation and guidelines be improved?

**Comment by Anusha000000** [comment]
Very good effort. Maybe guidelines can be presented to our partner organisations too for their feedback.

**Reply by lorrainewongmw** [reply]
When sharing the guidelines with partner organisations, perhaps we can invite them to post their feedback and countries of their work publicly on Git? Do you think they will be eager to comment here?

**Reply by Anusha000000** [reply]
Maybe, but I am not sure how many would be comfortable about using Github. We could probably keep option of them positing and a documentation posting open.

**Comment by OtiendeS** [comment]
I think my idea would be to think about this regionally. organizations are already forced to work together regionally and making use of already established regional coalitions to initiate, educate and give input to the standards will be helpful. We need to take advantage of already established communities.

**Reply by philhb** [reply]
@VerenaSattler

**Comment by Coconutchubby** [comment]
In this system: 
(1) Is there any case summary page so that the case manager can easily find with its case no. and give an update? 
(2) The option should be standardized and consistent because the user can drop down to see options rather type the content. It's also good for reporting as well.
(3) Is there any alert system to update the case or used for monitoring and evaluation purposes?

I'm really interested in this system. When does it go live?

**Reply by NoelPadalhin** [reply]
I agree on the second point of @Coconutchubby .  This makes for a shorter encoding time, which in turn makes for a user-friendly system. Governments may want to consider investing in continuous UX/UI of data systems. For the third point, what would be the standard alert time? Thoughts?
